packages/design-system/src/components/Pagination/Pagination.tsx

```diff
@@ -144,7 +144,7 @@ function Pagination({
   totalPages,
   ...rest
 }: PaginationProps): React.ReactElement {
-  const classes = classNames('ds-c-pagination', className);
+  const classes = classNames('ds-c-pagination', { 'ds-c-pagination__compact': compact }, className);
```

Comment:
To follow BEM syntax, this should be `ds-c-pagination--compact`. Variations or states are signified by `--`, whereas `__` is for child elements.

Reply:
oh, right! been a while since I've actually needed to create any classes.. Actually I think this is the first time I've touched CSS outside of token stuff since I started here believe it or not 😆